lib/app/error_reporting.dart

```diff
@@ -101,7 +100,7 @@ Future<bool> reportEvent(Event event) async {
 
   final packageInfo = await PackageInfo.fromPlatform();
   final platformString = defaultTargetPlatform.toString();
-  User user = HiveCache.isInitialized ? storage.userFromCache : null;
+  User user = await storage.userFromCache;
```

Comment:
`reportEvent` might be called before Hive is initialized. In that case, just calling `userFromCache` results in an exception:
```
E/flutter (19058): [ERROR:flutter/lib/ui/ui_dart_state.cc(157)] Unhandled Exception: NoSuchMethodError: The method 'get' was called on null.
E/flutter (19058): Receiver: null
E/flutter (19058): Tried calling: get("597f153a667a0659ed0b77ce")
E/flutter (19058): #0      Object.noSuchMethod  (dart:core-patch/object_patch.dart:53:5)
E/flutter (19058): #1      HiveCacheImpl._get 
package:hive_cache/src/cache.dart:113
E/flutter (19058): <asynchronous suspension>
E/flutter (19058): #2      LoadableId.loadFromCache 
package:hive_cache/src/utils.dart:23
E/flutter (19058): #3      StorageService.userFromCache 
package:schulcloud/…/services/storage.dart:35
E/flutter (19058): #4      reportEvent 
package:schulcloud/app/error_reporting.dart:103
E/flutter (19058): <asynchronous suspension>
E/flutter (19058): #5      _reportLogEvent 
package:schulcloud/app/error_reporting.dart:69
E/flutter (19058): #6      Logger.log 
package:logger/src/logger.dart:108
E/flutter (19058): #7      Logger.e 
package:logger/src/logger.dart:87
E/flutter (19058): #8      runWithErrorReporting.<anonymous closure> 
package:schulcloud/app/error_reporting.dart:4
```